src/PointwiseFunctions/GeneralRelativity/GeodesicEquation.hpp

```diff
@@ -0,0 +1,65 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace gr {
+
+/*!
+ * \brief First-order formulation of the geodesic equation for null geodesics
+ * that is suitable for ray tracing.
+ *
+ * This is the formulation of the geodesic equation that is originally used for
+ * ray tracing in \cite Bohn:2014xxa (Eq. (4)) and \cite Bohn:2016afc (Eq. (6)).
+ *
+ * For null rays with position $x^\mu$, proper time (or affine geodesic
+ * parameter) $\tau$, and four-momentum $p^\mu = dx^\mu / d\tau$, we first
+ * define the momentum variable
+ * \begin{equation}
+ *   \pi_i = \frac{p_i}{\alpha p^0} = \frac{p_i}{\sqrt{\gamma^{jk} p_j p_k}}
```

Comment:
Any particular reason for using `\pi` instead of the references' `\Pi`?  Not wrong, of course, but seems an odd change.

Reply:
fixed 👍